src/packages/core/components/input-start-node/input-start-node.element.ts

```diff
@@ -0,0 +1,94 @@
+import { UmbInputDocumentElement } from '@umbraco-cms/backoffice/document';
```

Comment:
It looks like all this input does is select between different content types. Maybe we could rename this to input-content-type instead? Then it could also be used for other things that needs to select between content types.

I also think this component should go into the content-type folder inside the core folder. You can see an example of how we have components in the culture folder.

Reply:
sounds good